storage/client/blob/src/main/java/com/azure/storage/blob/BlobClientBuilder.java

```diff
@@ -160,6 +180,10 @@ public AppendBlobAsyncClient buildAppendBlobAsyncClient() {
     }
 
     /**
+     * Creates a {@link BlockBlobClient} based on options set in the Builder. BlockBlobClients are used to perform block
```

Comment:
Can you modify it to "perform generic upload operations such as uploadFromFile in addition to block blob specific operations..."